services/movie_session.py

```diff
@@ -38,7 +38,15 @@ def update_movie_session(
     if cinema_hall_id:
         movie_session.cinema_hall_id = cinema_hall_id
     movie_session.save()
+    return movie_session
 
 
 def delete_movie_session_by_id(session_id: int) -> None:
     MovieSession.objects.get(id=session_id).delete()
```

Comment:
The `delete_movie_session_by_id` function uses `MovieSession.objects.get(id=session_id).delete()`, which will raise a `DoesNotExist` exception if no session with the given ID is found. Consider handling this exception to improve robustness.